sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/FlattenTest.java

```diff
@@ -130,8 +130,8 @@ public void testEmptyFlattenAsSideInput() {
 
     PCollection<String> output = p
         .apply(Create.of((Void) null).withCoder(VoidCoder.of()))
-        .apply(ParDo.withSideInputs(view).of(new OldDoFn<Void, String>() {
-                  @Override
+        .apply(ParDo.withSideInputs(view).of(new DoFn<Void, String>() {
+                  @ProcessElement
```

Comment:
Agree that it is a bit extreme. `google-java-format` makes it actually even more extreme, but I do think we should just standardize on that. Another time.